app/src/main/java/com/asfoundation/wallet/billing/adyen/AdyenPaymentFragment.kt

```diff
@@ -116,182 +119,145 @@ class AdyenPaymentFragment : BasePageViewFragment(), AdyenPaymentView {
   private var paymentDataSubject: ReplaySubject<AdyenCardWrapper>? = null
   private var paymentDetailsSubject: PublishSubject<AdyenComponentResponseModel>? = null
   private var adyen3DSErrorSubject: PublishSubject<String>? = null
-  private var isStored = false
-  private var askCVC = true
 
-  private val bindingCreditCardPreSelected: AdyenCreditCardPreSelectedBinding? by lazy {
-    if (isPreSelected) AdyenCreditCardPreSelectedBinding.bind(
-      requireView()
-    ) else null
-  }
   private val bindingCreditCardLayout: AdyenCreditCardLayoutBinding? by lazy {
-    if (!isPreSelected) AdyenCreditCardLayoutBinding.bind(
-      requireView()
-    ) else null
+    AdyenCreditCardLayoutBinding.bind(requireView())
   }
 
   // dialog_buy_buttons.xml
-  private val buy_button: WalletButtonView
-    get() = bindingCreditCardPreSelected?.dialogBuyButtonsPaymentMethods?.buyButton
-      ?: bindingCreditCardLayout?.dialogBuyButtons?.buyButton!!
-  private val cancel_button: WalletButtonView
-    get() = bindingCreditCardPreSelected?.dialogBuyButtonsPaymentMethods?.cancelButton
-      ?: bindingCreditCardLayout?.dialogBuyButtons?.cancelButton!!
+  private val buyButton: WalletButtonView
+    get() = bindingCreditCardLayout?.dialogBuyButtons?.buyButton!!
+  private val cancelButton: WalletButtonView
+    get() = bindingCreditCardLayout?.dialogBuyButtons?.cancelButton!!
 
```

Comment:
Once we don’t have two layouts in the same fragment anymore, I don’t see why to keep variables, why don’t call it directly as the rest of the app, like binding.dialogBuyButtons.cancelButton